syntax = "proto3";

package greet;

service Notifications {
	// Unary
	rpc GetUserDetails (UserDetailRequest) returns (UserDetailResponse);
}

message UserDetailRequest {
    int64 id = 1;
  }
  
 
  message UserDetailResponse {
    string first_name = 1;
    string last_name = 2;
    string email = 3;
  }